WolvenKit.App/ViewModels/Tools/ChunkViewModelTools.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using DynamicData.Binding;
+using WolvenKit.App.ViewModels.Shell;
+using WolvenKit.RED4.Types;
+
+namespace WolvenKit.App.ViewModels.Tools;
+
+public class ChunkViewModelTools
+{
+    public void SetChildExpansionStates(ChunkViewModel? node, bool isExpanded)
```

Comment:
public static void SetChildExpansionStates(this ChunkViewModel? node, bool isExpanded)